src/App.jsx

```diff
@@ -9,14 +9,14 @@ import {
 } from "react-router-dom";
 import "./App.scss";
 import Profile from "./Pages/Profile";
-import RootLayout from "./components/RootLayout/RootLayout";
+import Header from "./components/Header/Header";
```

Comment:
Why `Header/Header`?

Reply:
I've placed `Header.jsx` and `Header.scss` in the `Header` folder. Would you share how can I structure these files?

Reply:
@njm-wtag under Header folder rename the file from `header.jsx` to `index.jsx`